server/src/main/java/org/opensearch/index/mapper/RootObjectMapper.java

```diff
@@ -311,28 +313,22 @@ public DynamicTemplate[] dynamicTemplates() {
         return dynamicTemplates.value();
     }
 
-    @SuppressWarnings("rawtypes")
```

Comment:
This class is marked as `@PublicApi(since = "1.0.0")` so we have to preserve these methods